src/sentry/tasks/integrations/github/pr_comment.py

```diff
@@ -1,8 +1,25 @@
+from typing import List, Tuple
+
 from django.db import connection
 
 from sentry.models import GroupOwnerType
 
 
+def format_comment(issues: List[Tuple[str]]):
+    body = "## Suspect Issues\nThis pull request has been deployed and Sentry has observed the following issues:\n\n"
+
+    for issue in issues:
+        title = issue[0]
+        subtitle = issue[1]
+        formatted_subtitle = subtitle[:47] + "..." if len(subtitle) > 50 else subtitle
```

Comment:
string formatting allows for truncating
`{subtitle:.49}`

Reply:
do we want to truncate it without ellipses if it's too long?

Reply:
oh yeah, that's a good point